charts/cnpg-sandbox/README.md

```diff
@@ -34,4 +34,4 @@ A sandbox for CloudNativePG
 | kube-prometheus-stack.grafana.adminPassword | string | `"prom-operator"` | the grafana admin password |
 
 ----------------------------------------------
-Autogenerated from chart metadata using [helm-docs v1.10.0](https://github.com/norwoodj/helm-docs/releases/v1.10.0)
+Autogenerated from chart metadata using [helm-docs v1.11.0](https://github.com/norwoodj/helm-docs/releases/v1.11.0)
```

Comment:
I don't know  where does this come but if it goes in the cnp-sandbox it should go with that